<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="m2l1_kit_tray">
    <static>false</static>
    <allow_auto_disable>false</allow_auto_disable>

    <link name="m2l1_kit_tray">
      <inertial auto="true"/>
      
      <visual name="m2l1_kit_tray">
        <geometry>
          <mesh>
            <uri>model://demo_parts/m2l1_kit_tray/meshes/m2l1_kit_tray.stl</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1.0 1.0 1.0 1.0</ambient>
          <diffuse>1.0 1.0 1.0 1.0</diffuse>
        </material>
      </visual>

      <collision name="m2l1_kit_tray_collision">
        <density>20000</density>
        <pose>0 0 0 0 0 0</pose>
        <geometry>
          <mesh>
            <uri>model://demo_parts/m2l1_kit_tray/meshes/m2l1_kit_tray.stl</uri>
          </mesh>
        </geometry>
        <max_contacts>5</max_contacts>
      </collision>
    </link>
  </model>
</sdf>